plugins/importexport/native/filter/NativeXmlIssueFilter.inc.php

```diff
@@ -74,8 +73,21 @@ function handleElement($node) {
 		$issue->setCurrent($node->getAttribute('current'));
 		$issue->setAccessStatus($node->getAttribute('access_status'));
 
-		$issueDao->insertObject($issue);
-		$deployment->setIssue($issue);
+		$foundIssue = null;
+		foreach ($node->getElementsByTagName('issue_identification') as $n) {
+			$searchIssue = $issueDao->newDataObject();
+			$this->parseIssueIdentification($n, $searchIssue, false);
+			$foundIssues = $issueDao->getIssuesByIdentification($context->getId(), $searchIssue->getVolume(), $searchIssue->getNumber(), $searchIssue->getYear(), $searchIssue->getTitle(null));
+			foreach ($foundIssues->toArray() as $foundIssue) {
+				$deployment->addWarning(ASSOC_TYPE_ISSUE, $foundIssue->getId(), __('plugins.importexport.native.import.error.issueIdentificationDuplicate', array('issueId' => $foundIssue->getId(), 'issueIdentification' => $n->ownerDocument->saveXML($n))));
```

Comment:
I included this for two reasons:
1) it is a change from earlier behavior, where a new duplicate issue with the same identification would be created
2) any attributes of the issue added/modified by this import will *not* be rolled back in the event of error.